app/views/staches/show.html.erb

```diff
@@ -22,7 +22,8 @@
          <p><%= @stache.description %></p>
        </div>
        <div class="card-action">
-         <%= link_to "Add to Cart", cart_staches_path(stache_id: @stache.id), method: :post %>
+         <%= link_to("Add to Cart", cart_staches_path(stache_id: @stache.id),
+         method: :post) unless @stache.retired %>
```

Comment:
Can we use the ? mark version of this method to keep with the ruby convention for returning booleans?